students/junghyun/users/views.py

```diff
@@ -43,3 +39,21 @@ def post(self,request):
         
         except DataError:
             return JsonResponse({"message":"DATA_ERROR"}, status=401)
+        
+class SignInView(View):
+    def post(self, request):
+        data = json.loads(request.body)
+        try:
+            password = User.objects.get(email = data['email']).password
+            if (password != data['password']):
+                raise ValidationError("INVALID_PASSWORD")
+            return JsonResponse({'message':'SUCCESS'},status = 200)
+        
+        except ValidationError as e:
+            return JsonResponse({'message':e.message}, status = 401)
+        
+        except User.DoesNotExist:
+            return JsonResponse({'message':'INVALID_EMAIL'}, status=401)
```

Comment:
이 에러는 정현님 로직상 이메일만 판단해서 나오는 에러가 아닙니다! 에러 메세지를 조금 더 명확하게 적어주세요!
```suggestion
        except User.DoesNotExist:
            return JsonResponse({'message':'invalid_user'}, status=401)
```

Reply:
네 감사합니다 수정했습니다~!~!~!